influxdb/client.py

```diff
@@ -495,6 +495,32 @@ def drop_database(self, dbname):
         """
         self.query("DROP DATABASE {0}".format(quote_ident(dbname)))
 
+    def get_list_measurements(self):
+        """Get the list of measurements in InfluxDB.
+
+        :returns: all measurements in InfluxDB
+        :rtype: list of dictionaries
+
+        :Example:
+
+        ::
+
+            >> dbs = client.get_list_measurements()
+            >> dbs
+            [{u'name': u'measurements1'},
+             {u'name': u'measurements2'},
+             {u'name': u'measurements3'}]
+        """
+        return list(self.query("SHOW MEASUREMENTS").get_points())
+
+    def drop_measurement(self, measurementname):
+        """Drop a measurement from InfluxDB.
+
+        :param dbname: the name of the measurement to drop
```

Comment:
Should be `measurementname` instead of `dbname` according to the method's signature. But even better would be to rename the parameter to `measurement` in order to be consistent with naming in other methods.